flowz/channels/tools.py

```diff
@@ -34,15 +34,15 @@ def rolling(window_size):
     return Roller(window_size)
 
 
-def pinned_group_size(lower=0, upper=sys.maxint):
+def pinned_group_size(lower=0, upper=sys.maxsize):
```

Comment:
I double-checked the docs to be sure and agree that maxsize is the right thing to do.